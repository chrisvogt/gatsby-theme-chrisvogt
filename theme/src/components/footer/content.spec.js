import React from 'react'
import renderer from 'react-test-renderer'
import Content from './Content'
import Profiles from './profiles'

jest.mock('./profiles')

describe('Footer/Content', () => {
  Profiles.mockImplementation(() => <div className='MOCK__Profiles'></div>)
  it('matches the snapshot', () => {
    const tree = renderer
      .create(
        <Content
          footerText='Â© 2019-2020 Chris Vogt'
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
