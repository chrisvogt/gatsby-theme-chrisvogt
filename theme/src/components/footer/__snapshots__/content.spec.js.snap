// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer/Content matches the snapshot 1`] = `
<div
  className="css-o91fde-Content"
>
  <div
    className="css-f4963r-Content"
  >
    <div
      className="MOCK__Profiles"
    />
  </div>
  <div>
    Â© 2019-2020 Chris Vogt
  </div>
</div>
`;
